frontend/src/components/hotel-form/index.tsx

```diff
@@ -23,6 +23,7 @@ type Props = {
     checkout: moment.Moment,
   ) => void;
   removeHotelRoom: (id: string, index: number) => void;
+  me: CurrentUserQueryResult["data"]["me"];
```

Comment:
Why you can't use the dot notation here? `CurrentUserQueryResult` is a type, isn't it? 🤔

Reply:
Uhm can we? This is the typescript definition, not code, basically we are telling TS here that the type of `me` is the type of `CurrentUserQueryResult > data > me`

<img width="786" alt="image" src="https://user-images.githubusercontent.com/3382153/147392889-18c615e5-035c-4422-a36a-cc7866b573c3.png">

Reply:
https://www.typescriptlang.org/play?ssl=9&ssc=1&pln=9&pc=2#code/FAFwngDgpgBAgjAvDA3sGMCGAuGBnEAJwEsA7AcwG5gBfYUSWAISVXRhCgNzgG0AiTPwC61DJwIAmHgDpMtIA

Doesn't seem to be supported